### DISCUSSION TITLE
Ring of Seven Curses

### DISCUSSION THREAD

**Discussion by MisterMeddler** [Aizistral-Studios/Enigmatic-Legacy#135] [Q&A] [answered]
How do I find this or acquire this? No wiki entry for it that I can find, is that intentional? I see so many references to this but have no idea how to get the dang thing let alone cure the curses which seems to be a thing?

**Comment by Aizistral** [comment, marked as answer]
By default, it is added to your inventory when you enter a new world for the first time.

**Comment by Sunnyshine11630** [comment]
where can i go to see why my pack isn't adding it by default? I'm playing better minecraft plus and want the ring but dont want to just crawl into creative... also im on a server that doesn't do creative/cheats so im hoping by me figuring out how to get it that I can help them get it in the pack. I can't find it in loot tables either.

**Reply by Sunnyshine11630** [reply]
@Aizistral

**Reply by Aizistral** [reply]
Since this is the fourth time people come to me from that Better Minecraft Plus thing, this time I'll just go ahead and add notice on CurseForge page:
![image](https://user-images.githubusercontent.com/47505981/135182715-98e48cc0-2030-4bb7-9295-e1a5566fe6e1.png)
